assets/models/system/stabilityai-stable-diffusion-xl-base-1.0/spec.yaml

```diff
@@ -0,0 +1,29 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: stabilityai-stable-diffusion-xl-base-1.0
+path: ./
+properties:
+  SharedComputeCapacityEnabled: true
+  SHA: 462165984030d82259a11f4367a4eed129e94a7b
+  inference-min-sku-spec: 6|1|112|336
+  inference-recommended-sku: Standard_NC6s_v3, Standard_NC12s_v3, Standard_NC24s_v3,
+    Standard_ND40rs_v2, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4
+tags:
+  Preview: ""
+  SharedComputeCapacityEnabled: ""
+  license: creativeml-openrail++-m
+  task: text-to-image
+  huggingface_model_id: stabilityai/stable-diffusion-xl-base-1.0
+  author: stabilityai
+  training_dataset: LAION-5B
+  inference_supported_envs:
+  - ds-mii
```

Comment:
we need to verify ds-mii. please confirm with Gaurav

Reply:
ds-mii is not working due to following 2 issues:
1. prod image for deepspeed is not accessible due to some issue
2. when we tested against the local docker image on workspace, we observed that the inference code is different for ds-mii and it doesn't validate teh nsfw_content_detected attribute which is not available for xl pipelines from HF

will address separatel in next round!